modules/dnn/src/op_inf_engine.cpp

```diff
@@ -462,14 +474,25 @@ void InfEngineBackendNet::initPlugin(InferenceEngine::ICNNNetwork& net)
             // Some of networks can work without a library of extra layers.
 #ifndef _WIN32
             // Limit the number of CPU threads.
+#if INF_ENGINE_VER_MAJOR_LE(INF_ENGINE_RELEASE_2019R1)
             enginePtr->SetConfig({{
                 InferenceEngine::PluginConfigParams::KEY_CPU_THREADS_NUM, format("%d", getNumThreads()),
             }}, 0);
+#else
+            if (device_name == "CPU")
+                core.SetConfig({{
+                    InferenceEngine::PluginConfigParams::KEY_CPU_THREADS_NUM, format("%d", getNumThreads()),
+                }}, device_name);
+#endif
 #endif
         }
+#if INF_ENGINE_VER_MAJOR_LE(INF_ENGINE_RELEASE_2019R1)
         plugin = InferenceEngine::InferencePlugin(enginePtr);
-
         netExec = plugin.LoadNetwork(net, {});
+#else
+        std::shared_ptr<InferenceEngine::ICNNNetwork> cnn_net(&net, [] (InferenceEngine::ICNNNetwork*) {});
+        netExec = core.LoadNetwork(InferenceEngine::CNNNetwork(cnn_net), device_name);
```

Comment:
Please replace `initPlugin`'s input from ICNNNetwork to CNNNetwork as in here: https://github.com/opencv/opencv/pull/15063/files#diff-da37b66e77aa9907ebc193c732dd00d1R95

It'll help to use just `netExec = core.LoadNetwork(net)`;